frame/staking-rewards/src/lib.rs

```diff
@@ -452,6 +506,8 @@ pub mod pallet {
 		type RewardPoolId = T::RewardPoolId;
 
 		#[transactional]
+		// Review [Andy]
+		// could the naming be better, we create a staking pool but return a reward pool
```

Comment:
staking and reward pool are used somewhat interchangably here (unfortunately), the whole pallet could be reviewed in that regard